randovania/games/planets_zebeth/exporter/game_exporter.py

```diff
@@ -42,8 +49,51 @@ def export_params_type(self) -> type[GameExportParams]:
     def _do_export_game(
         self,
         patch_data: dict,
-        export_params: GameExportParams,
+        export_params: PlanetsZebethGameExportParams,
```

Comment:
Don't do this.
Keep it as GameExportParams, and then make an isinsance assertion on whether its a zebeth one.